### ISSUE TITLE
Can't resolve lodash.merge

### ISSUE THREAD

**Issue by Async0x42** [MatteoGabriele/vue-analytics#6]
I'm getting the following error upon importing, with the basic auto-tracking config:

```
ERROR in ./~/vue-analytics/dist/vue-analytics.js
Module not found: Error: Can't resolve 'lodash.merge' in '...\node_modules\vue-analytics\dist'
 @ ./~/vue-analytics/dist/vue-analytics.js 7:90-113
```

**Comment by MatteoGabriele** [comment]
Could you please provide me an example or better a little repo with your issue, I can't really reproduce it.

**Comment by louisbourque** [comment]
I was getting this error too.

I was able to fix it by updating the package.json file. I moved `"lodash.merge": "^4.6.0"` from `devDependencies` to `dependencies`. Then I did an npm install.

It looks like when doing an npm install, only dependencies are installed. If you happen to have lodash.merge installed in that repo or globally, it'll work.

**Comment by MatteoGabriele** [comment]
Just published v2.0.2
